src/Tablo/Cell/Cell.tsx

```diff
@@ -0,0 +1,95 @@
+import * as React from 'react';
+import * as cx from 'classnames';
+import { props, t, ReactChildren } from '../../utils';
+import FlexView from 'react-flexview';
+import { Cell as CellFDT } from 'fixed-data-table-2';
+import { If, StringContains, StringIntersection } from 'typelevel-ts';
+
+export type PickIfExists<T, K extends string> = If<StringContains<keyof T, K>, T[StringIntersection<keyof T, K>], undefined>;
+
```

Comment:
wouldn't be simply
```
export type PickIfExists<T, K extends string> = If<StringContains<keyof T, K>, T[K], undefined>;
```
?

Reply:
Also, I don't get what would happen in these cases
![image](https://user-images.githubusercontent.com/3280300/31025522-39d80d1e-a543-11e7-8abc-06ff269b789e.png)

Reply:
> wouldn't be simply

Unfortunately not: in `If<a, b, c>`, `K` cannot yet be used in `b` position as `T[K]` since `K extends keyof T` is not true there. `StringIntersection` makes it true